modules/openapi-generator/src/main/resources/ruby-client/api_client_faraday_partial.mustache

```diff
@@ -10,8 +10,10 @@
         :client_cert => @config.ssl_client_cert,
         :client_key => @config.ssl_client_key
       }
-
-      connection = Faraday.new(:url => config.base_url, :ssl => ssl_options) do |conn|
+      request_options = {
+        :params_encoder => @config.params_encoder
+      }
+      connection = Faraday.new(:url => config.base_url, :ssl => ssl_options, :request => request_options) do |conn|
```

Comment:
minor suggestion: what about declaring the hash inline without declaring a variable `request_options` to store it?

Reply:
Either way works for me. I'll update to be inline. 

I can't find instructions for how to regenerate the samples. Can you please remind me?

Reply:
Nevemind ... I found the command in the failed CI run.

Reply:
I saw you pushed some changes but don't any changes to this file. Did you commit the change before the push?

Reply:
I decided not to make the change because the line was very long. 

I ran the script to update the samples, but it seems like there are more changes that it didn't pick up. Do you know what I am doing wrong?

Reply:
That's ok. Let's see if someone from the community can help make that enhancement later.

I've updated the samples via https://github.com/OpenAPITools/openapi-generator/pull/9839/commits/46cd6b8cf679a6a75db1fb5a7989b316e5b5643a